app/src/main/java/io/github/droidkaigi/confsched2018/presentation/common/activity/BaseActivity.kt

```diff
@@ -11,4 +16,17 @@ open class BaseActivity : AppCompatActivity() {
     override fun attachBaseContext(newBase: Context) {
         super.attachBaseContext(CalligraphyContextWrapper.wrap(newBase))
     }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        initTaskDescription()
+    }
+
+    private fun initTaskDescription() {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+            setTaskDescription(ActivityManager.TaskDescription(
+                    null, null, ContextCompat.getColor(this, R.color.recents_background))
```

Comment:
1st `null` is task label, but we don't change it currently.
2nd `null` is task icon, but we don't change it currently.